src/social/SocialIcon.js

```diff
@@ -141,7 +144,7 @@ const SocialIcon = props => {
 
 SocialIcon.propTypes = {
   Component: PropTypes.func,
-  type: PropTypes.string,
+  name: PropTypes.string,
```

Comment:
may also be a good idea here to have like,

PropTypes.oneOf([ 'facebook']) etc. Since we actually don't accept "any string" the user passes

Reply:
should I copy and paste it from the docs?
```
social media type (angellist, codepen, envelope, etsy, facebook,
foursquare, github-alt, github, gitlab, instagram, linkedin, medium,
pinterest, quora, reddit-alien, soundcloud, stack-overflow,
steam, stumbleupon, tumblr, twitch, twitter,
google-plus-official, vimeo, wordpress, youtube)
```

Reply:
Yea, will need to do it for typescript as well